flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -989,6 +990,8 @@ void OmpStructureChecker::Enter(const parser::OmpClause::Schedule &x) {
   }
 }
 
+void OmpStructureChecker::Enter(const parser::OmpClause::ScheduleModifier &x) {}
```

Comment:
Is this a "do nothing"? or is some code needed here eventually?

Either way, please add a comment to clarify it to readers.

Reply:
To the best of my understanding, there is nothing that needs to be done here, but the function must exist.

Reply:
There should be at least a `CheckAllowed(llvm::omp::Clause::OMPC_schedule_modifier);` here.

Reply:
Ah, I will update, thanks.

Reply:
Actually removed this code, as it is not a clause any more, and there's nothing to be done here.